### ISSUE TITLE
TSC Meeting #12 2019-07-03

### ISSUE THREAD

**Issue by rudygalfi** [ampproject/meta-tsc#15]
# Agenda

* [Resolved via email] Finalize AC member attendance plan raised via email

@ampproject/tsc
Designated AC attendees: @cvazac @kenjibaheux

**Comment by rudygalfi** [comment]
This meeting is scheduled close to a US holiday. I'm curious if we'll have sufficient attendance to keep it or if we should plan to cancel.

**Comment by davidstrauss** [comment]
> This meeting is scheduled close to a US holiday. I'm curious if we'll have sufficient attendance to keep it or if we should plan to cancel.

I will definitely be in the office on July 3rd, but I understand if it's an inconvenient date for others.

**Comment by rudygalfi** [comment]
> Finalize AC member attendance plan raised via email

Regarding this, we resolved this over email: We decided that TSC members will manually add the relevant AC members onto the meeting invitation. I will start the process of indicating the attending AC members as part of these agenda issues, see above ^^. Whoever creates the agenda issues should also ensure the AC members listed matches the roster.

Regarding this July 3 meeting, I will be unavailable. Seeing no additional topics to cover on our agenda at this time, I suggest canceling.

cc @cvazac @kenjibaheux
